### PR TITLE
Add VideoTransformDevice interface stub

### PR DESCRIPTION
**Pull request by XHatan**
**Issue #:**

**Description of changes:**

Add `VideoTransformDevice` interface stub.
**Testing**

1. Have you successfully run `npm run build:release` locally?
2. How did you test these changes?
3. Can these changes be tested using the demo application? If yes, which demo application can be used to test it?
4. Do you add, modify, or delete public API definitions? If yes, has that been reviewed and approved?
5. Do you change the wire protocol, e.g. the request method? If yes, has that been reviewed and approved?

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Review by anuranduttaroy — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 ### Added
+- Add VideoTransformDevice interface stub
```
Comment (richnew10) [inline CHANGELOG.md:10]: I would phrase this as 

```
- Add `VideoTransformDevice` interface placeholder. This interface mirrors `AudioTransformDevice`. Choosing `VideoTransformDevice`s in `DefaultDeviceController` will be implemented in a future release.
```

and move it down below line 13 to contextualize with `AudioTransformDevice`. This is important guidance for builders, so it's worth being verbose.

File: test/devicecontroller/DefaultDeviceController.test.ts [hunk 2]
```diff
@@ -4,6 +4,7 @@
 import * as chai from 'chai';
 import * as sinon from 'sinon';
 
+import { Device, VideoTransformDevice } from '../../src';
```
Comment (richnew10) [inline test/devicecontroller/DefaultDeviceController.test.ts:7]: These, ironically, we prefer to import from the individual source files.

File: src/devicecontroller/VideoTransformDevice.ts [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+   */
+  readonly outputMediaStream: MediaStream;
+}
+
+/**
+ * `isVideoTransformDevice` is a type guard for {@link VideoTransformDevice}.
+ *
+ * @param device the value to check.
+ */
+export function isVideoTransformDevice(device: unknown): device is VideoTransformDevice {
```
Comment (richnew10) [inline src/devicecontroller/VideoTransformDevice.ts:37]: You will need to explicitly add this to `barrelize.js`; see where I did this for `isAudioTransformDevice` on line 60.

**Review by richnew10 — CHANGES_REQUESTED** [review]

File: test/devicecontroller/DefaultDeviceController.test.ts (hunk 2 above)
Comment (XHatan) [reply test/devicecontroller/DefaultDeviceController.test.ts]: Fixed.

File: src/devicecontroller/VideoTransformDevice.ts (hunk 3 above)
Comment (XHatan) [reply src/devicecontroller/VideoTransformDevice.ts:37]: Added explicitly.

File: CHANGELOG.md (hunk 1 above)
Comment (XHatan) [reply CHANGELOG.md]: Sounds good.

**Review by richnew10 — APPROVED** [review]

**Pull request merged**